net-mail/gnubiff/gnubiff-2.2.17-r3.ebuild

```
@@ -0,0 +1,47 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit autotools
+
+DESCRIPTION="A mail notification program"
+HOMEPAGE="http://gnubiff.sourceforge.net/"
+SRC_URI="https://downloads.sourceforge.net/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-3+-with-openssl-exception"
+SLOT="0"
+KEYWORDS="amd64 x86"
```

```suggestion
KEYWORDS="~amd64 ~x86"
```
You cannot commit straight to stable.